ChartIQ/finsemble-dotnet-seed
# 14869 Add get and set component state examples

https://chartiq.kanbanize.com/ctrl_board/48/cards/14869/details/

PR for DLL changes required for it to build (already added to Finsemble resources folder for testing): https://github.com/ChartIQ/finsemble-dll/pull/28

Exposes WindowClient in the DLL so that it can be used by clients and adds an example of use to the WPFExample.

Testing:
- Build the WPFExample and setup for use in Finsemble
- set Symbol in input field to somethign other than default
- perform an action that will cause the state to save (doesn't save on entry, but rather on Drag or hit of the send button)
- save into workspace and reload, observe correct symbol is still set
- use another component to change the symbol via a linker colour channel and repeat workspace save test
- use another component to change the symbol via a drag and drop action and repeat workspace save test
